mpl_volume_viewer.py

```diff
@@ -107,12 +107,12 @@ def process_mouse_button(self, event):
             final_dim = self.index[dim]
             point.insert(dim, final_dim)
             point = [int(round(i)) for i in point]
-            self.set_viewpoint(point)
+            self.set_viewpoint(point, [1,2] if dim == 0 else [0,2] if dim == 1 else [0,1] if dim == 2)
```

Comment:
```suggestion
            self.set_viewpoint(point, list({0, 1, 2} - {dim}))
```

... for example :smiley: